### PR TITLE
Allow run on PR(s), make it callable.

### PR DESCRIPTION
**Pull request by niedbalski**
- Allow run on PR(s), make the integration test a callable.
- Rename the performance workflows.

Related to #54 

Signed-off-by: Jorge Niedbalski <j@calyptia.com>

### CODE REVIEW

**Review by patrick-stephens — APPROVED** [review]

File: .github/workflows/ci.yaml
Comment (niedbalski) [inline .github/workflows/ci.yaml:20]: this needs rename after approval.

**Pull request merged**
